lib/daru/vector.rb

```diff
@@ -884,40 +884,29 @@ def to_s
     end
 
     # Create a summary of the Vector using Report Builder.
-    def summary(method=:to_text)
-      ReportBuilder.new(no_title: true).add(self).send(method)
-    end
-
-    # :nocov:
-    def report_building b # rubocop:disable Metrics/AbcSize,Metrics/MethodLength
-      b.section(name: name) do |s|
-        s.text "n :#{size}"
-        s.text "n valid:#{count_values(*Daru::MISSING_VALUES)}"
-        if @type == :object
-          s.text  "factors: #{factors.to_a.join(',')}"
-          s.text  "mode: #{mode}"
-
-          s.table(name: 'Distribution') do |t|
```

Comment:
Well, the idea of "drop reportbuilder" is to make the list of dependencies smaller. Will text-table make the code significantly simpler? And if so, is it the better "table formatting" gem selection? It was not updated for two years.